wordpress-mobile/WordPress-Android
# Jetpack App Basics: Create resource directories

Fixes #14297 

This PR adds the following res directories and initial app icons to the `feature/jetpack-app-basics` feature branch:
- icons directories (drawable, mipmap...).
- xml directory (shortcuts.xml)

**Merge instructions**
- Ensure that PR #14320 has been merged
- Remove "Not ready for merge" label
- Merge as normal

**Pre test**
- You must update your local copy of `google-services.json` before running the app.
-- Copy existing `org.wordress.android`,  `org.wordress.android.prealpha`, `org.wordress.android.beta` sections and replace with `com.jetpack.android`, `com.jetpack.android.prealpha`, `com.jetpack.android.beta`.

**To Test**
- Build and launch each product flavor. Verify the build variant launches.
- `wordpressVanillaDebug` (Prod)
- `wordpressWasabiDebug` (Beta)
- `wordpressJalapenoDebug` (Alpha)
- `jetpackVanillaDebug` (Prod)
- `jetpackWasabiDebug` (Beta)
- `jetpackJalapenoDebug` (Alpha)

Note that:
- `jetpackDebug` should build all `debug` product flavors related to the `jetpack` dimension.
- `jetpackJalapeno` should build both, the `debug` and `release`, `jalapeno` product flavor for `jetpack` dimension.
- `jetpackWasabi` should build both, the `debug` and `release`, `wasabi` product flavor for `jetpack` dimension.

Open App info and verify that the Icons and App names match the following image:
<img src="https://user-images.githubusercontent.com/506707/112046192-17d64780-8b22-11eb-8dfc-5c02ee2f1c60.png" width="350">

PR submission checklist:
